src/persistence/idpay.ts

```diff
@@ -418,9 +423,25 @@ range(0, walletConfig.discountCount).forEach(() => {
     ...instruments,
     [initiativeId]: []
   };
+
   initiativeTimeline = {
     ...initiativeTimeline,
     [initiativeId]: [
+      {
+        ...generateRandomOperationDTO(TransactionOperationEnum.TRANSACTION),
+        status: TransactionStatusEnum.AUTHORIZED,
+        channel: TransactionChannelEnum.QRCODE
+      } as OperationListDTO,
+      {
+        ...generateRandomOperationDTO(TransactionOperationEnum.TRANSACTION),
+        status: TransactionStatusEnum.REWARDED,
+        channel: TransactionChannelEnum.QRCODE
+      } as OperationListDTO,
+      {
+        ...generateRandomOperationDTO(TransactionOperationEnum.TRANSACTION),
+        status: TransactionStatusEnum.CANCELED,
```

Comment:
```suggestion
        status: TransactionStatusEnum.CANCELLED,
```
see https://github.com/pagopa/cstar-infrastructure/pull/949
it's a small typo, will be merged soon

Reply:
Will be fixed in next PR